packages/components/src/button/src/useToggleButton.ts

```diff
@@ -43,7 +43,8 @@ export function useToggleButton({
 
     return {
         buttonProps: {
-            active: active || isChecked,
+            active: active,
```

Comment:
double check if this change is expected or no. You don't seem to use those values